application/src/main/java/org/opentripplanner/apis/vectortiles/DebugStyleSpec.java

```diff
@@ -181,9 +197,8 @@ private static List<StyleBuilder> edges(VectorSourceLayer edges) {
         .group(EDGES_GROUP)
         .typeLine()
         .vectorSourceLayer(edges)
-        .lineColor(MAGENTA)
-        .edgeFilter(EDGES_TO_DISPLAY)
```

Comment:
The style changes have been reverted.

Since only the edge layer was _interactive_, my thought was to have an indicative background layer on which any edge could be selected.

That has been replaced with an update to `MapControl`/`LayerControl` so that the list of dynamic layers is updated based on the selected layers.